workspace-hack/Cargo.toml

```diff
@@ -26,6 +26,7 @@ aho-corasick = { version = "1" }
 anyhow = { version = "1" }
```

Comment:
Does this file need to be updated again, after the federated-dev re-removal?

Reply:
Yeah, done